packages/schematics/angular/utility/project.ts

```diff
@@ -35,6 +43,23 @@ export function getProject<TProjectType extends ProjectType = ProjectType.Applic
   return workspace.projects[projectName] as WorkspaceProject<TProjectType>;
 }
 
+export function getProjectConfig(host: Tree, path: string) {
+  const configName = path.split('/').pop();
+  const configFile = host.read(path);
+  if (!configFile) {
+    throw new Error(`Couldn't find project config (${configName}).`);
+  }
+  const configText = configFile.toString('utf-8');
+  let config: ProjectConfig;
+  try {
+    config = JSON.parse(configText || '{}');
```

Comment:
TS config can contain comments,
 
You should use the JSON parser.

Reply:
Thanks for the catch!